packages/mobile/src/screens/edit-track-screen/screens/CoverAttributionScreen.tsx

```diff
@@ -0,0 +1,74 @@
+import { useField } from 'formik'
+
+import { Box, Divider, Flex } from '@audius/harmony-native'
+import { Switch, Text } from 'app/components/core'
```

Comment:
can you try using harmony `Text` instead? and if there's a harmony `Switch` that as well.
basically no new UI code should use anything from `app/components/core` if it can be avoided.